platformio.ini

```
@@ -13,7 +13,7 @@
 [platformio]
 src_dir      = Marlin
 boards_dir   = buildroot/share/PlatformIO/boards
-default_envs = mega2560
+default_envs = HC32F460xCxx_AQUILA_V101
```

```suggestion
default_envs = mega2560
```

i'd like to keep the marlin source code as close to upstream as possible, so please revert this change